src/math/Box2.js

```diff
@@ -129,27 +129,13 @@ Box2.prototype = {
 
 	containsPoint: function ( point ) {
 
-		if ( point.x < this.min.x || point.x > this.max.x ||
```

Comment:
It's not my favorite hobbit !!! ^^
I just try to keep code consistency between files accordingly (from yesterday, great decision to implement this tool from zz85) to the new eslint module. And your code style.

In fact, there is a lot of improvement that could be done in my view (to keep new and older programmer from a lot of pain) but, for now, i do not know what are your goal for three.js... That's my problem !

What are your plan for the next ? Es6 class and module as i can see in [#9562](https://github.com/mrdoob/three.js/issues/9562#issuecomment-261006169) ? 
I suppose, in concordance with what i read from our exchanges, that:

- you want better code (of course)
- the code still remain readable for new comer (not too more complicated or obfuscated)
- backward compatibility was keep (At least until the next major release)
- the doc was up to date (but without using auto generation from comment in code source that pollute readability).
- you would remove bower dependency
- you would keep a single configuration file for all the project

If there are other please tell me, that will avoid me to ask for PR that will be rejected for not known yet reasons !